app/forms/collection_edit_form.rb

```diff
@@ -21,6 +27,8 @@ def self.model_attributes(attrs)
     attrs.dup.each do |key, value|
       if value.is_a? Array
         attrs[key].map! { |v| to_hash(v) }
+      elsif value.nil? || value.to_s.strip.blank?
```

Comment:
Oh, how interesting. I wonder if the nil check (or compact alternative that seems to be problematic) is actually needed since you're calling [`super`](https://github.com/projecthydra/hydra-editor/blob/4da9c0ea542f7fde512a306ec3cc90380691138b/app/forms/hydra_editor/form.rb#L58-L64) which appears to sanitize/delete empty attributes?

That said, you've spent the time on this code, and this isn't worth dwelling on too much. It was just a thought. Don't want to unnecessarily slow down this PR going in :)